generate.py

```diff
@@ -380,7 +380,6 @@ def generate(args):
     parser.add_argument('--gc-power', type=int, default=530, help='set power at grid connection '
                                                                   'point in kW')
     parser.add_argument('--seed', default=None, type=int, help='set random seed')
-
```

Comment:
This empty line was in there to separate the block with more specific arguments for generate and the file arguments. It doesn't hurt, makes it a bit more structured and this is the only change in this file. I'd say revert this change, so there is one file less in the diff.